template/en/default/bug/create/user-message.html.tmpl

```diff
@@ -30,7 +30,7 @@
 [% PROCESS global/variables.none.tmpl %]
 
 Before reporting [% terms.abug %], please read the 
-<a href="page.cgi?id=bug-writing.html">
```

Comment:
Is this already a broken link, or is that page removed by this patch as well?

Reply:
Not broken so far, but has been moved to MDN. Used the same URL as other guideline links.